logstash-core/build.gradle

```diff
@@ -109,12 +109,14 @@ dependencies {
     compile 'org.apache.logging.log4j:log4j-core:2.6.2'
     compile 'com.fasterxml.jackson.core:jackson-core:2.7.4'
     compile 'com.fasterxml.jackson.core:jackson-databind:2.7.4'
+    compile 'com.fasterxml.jackson.core:jackson-annotations:2.7.4'
```

Comment:
Yup...it's a race to which one will conflict first. Since the other Issue is blocked by a Jenkins build issue, this will probably go cleanly (first).